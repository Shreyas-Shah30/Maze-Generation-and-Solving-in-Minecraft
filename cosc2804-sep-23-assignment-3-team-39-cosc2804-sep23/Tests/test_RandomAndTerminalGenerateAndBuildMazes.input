1
2
3965 65 4205
3 7
2
1
1
3961 65 4198
5 9
xxxxxxxxx
........x
xxxxxxx.x
x.......x
xxxxxxxxx
2
6
