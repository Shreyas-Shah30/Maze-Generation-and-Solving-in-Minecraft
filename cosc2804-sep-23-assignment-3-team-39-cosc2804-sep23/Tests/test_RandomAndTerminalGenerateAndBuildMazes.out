
Welcome to MineCraft MazeRunner!
--------------------------------

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of the maze:
Enter the length and width of the maze:
Maze generated successfully
**Printing Maze**
BasePoint: (3965, 65, 4205)
Structure:
xxxxxxx
......x
xxxxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of the maze:
Enter the length and width of the maze:
Enter the maze structure:
Maze read successfully
**Printing Maze**
BasePoint: (3961, 65, 4198)
Structure:
xxxxxxxxx
........x
xxxxxxx.x
x.......x
xxxxxxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 
The End!

